script/management_graphs.Rmd

```diff
@@ -0,0 +1,209 @@
+---
+title: "Management_graphs"
+author: "Sander Devisscher"
+date: '2022-08-17'
+output:
+  word_document: default
+  html_document: default
+---
+
+```{r setup, include=FALSE}
+library(rgbif)
+library(geojsonsf)
+library(sf)
+library(tidyverse)
+```
+
+```{r, include=FALSE}
+cleanData <- read_csv("./interim/Lithobates_catesbeianus_management_data.csv")
+```
+
+# GRAPH: Total number of bullfrogs caught
+```{r total individual caught, include=FALSE}
+total_by_year <- cleanData %>% 
+  group_by(year) %>% 
+  summarise(n = sum(individualCount, na.rm = TRUE)) %>% 
+  ggplot(aes(x = year, y = n)) +
+  geom_bar(stat = "identity")
+```
+
+```{r echo=FALSE}
+plot(total_by_year)
+```
+
+# GRAPH: Absolute number of bullfrogs caught by lifestage
+```{r total individuals caught by lifestage, include=FALSE}
+total_by_year_by_lifestage <- cleanData %>% 
+  group_by(year, lifeStage) %>% 
+  summarise(n = sum(individualCount, na.rm = TRUE)) %>% 
+  filter(!is.na(lifeStage)) %>% 
+  ggplot(aes(x = year, y = n, fill = lifeStage)) +
+  geom_bar(stat = "identity",
+           position = "stack")
+```
+
+```{r echo=FALSE}
+total_by_year_by_lifestage
+```
+
+# GRAPH: Cumulative number of bullfrogs caught
+```{r include=FALSE}
+cum_by_year <- cleanData %>% 
+  arrange(year) %>% 
+  mutate(cum = cumsum(individualCount)) %>% 
+  group_by(year) %>% 
+  summarise(n = max(cum)) %>% 
+  ggplot(aes(x = year, y = n)) +
+  geom_bar(stat = "identity")
+```
+
+```{r echo=FALSE}
+plot(cum_by_year)
+```
+
+
+# GRAPH: Catch per unit of effort
+```{r total individuals caught by unit of effort, include=FALSE}
+total_by_year_corrected <- cleanData %>% 
+  group_by(year) %>% 
+  summarise(effort = n_distinct(eventID),
```

Comment:
Best wordt ergens vermeld (in grafiektitel, astitel, hoofding, toelichting...) wat 1 eenheid van inspanning betekent (1 bezoek/vangst/campagne in het veld), want zonder code is dat nu niet duidelijk. Bv. "_(1 unit = 1 field visit)_" of "_(1 unit = 1 netting action)_" ...

Reply:
@bramdhondt ik ga kijken of ik dit nog kan herwerken naar de effectieve definitie van een vpei die we altijd gebruiken aka 1 fuik voor 24u (dit komt codewise neer op huidige effort * #fuiken)